indico/queries/submission.py

```diff
@@ -399,3 +399,43 @@ def __init__(
     def process_response(self, response) -> Job:
         response = super().process_response(response)["submitAutoReview"]
         return Job(id=response["jobId"])
+
+
+class RetrySubmission(GraphQLRequest):
+    """
+    Given a list of submission ids, retry those failed submissions.
```

Comment:
does this check that the IDs past to it actually failed?

Reply:
The backend does check and returns an error string if the ID is already done